### PR TITLE
Tasks structure

### PR DESCRIPTION
**Pull request by viktor-yakubiv**
First attempt to create a structure of tasks. Refers to #90 

File names a partially tagged like `git-`, `html-css-`, `app-design-`.

Also, headers and basic structure are unified.

#### Next steps:

1. Add contents to ReadMe
2. Add `tasks/README.md` to describe the directory

#### Also plan:

1. Review task texts from 1 to 10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ You will see the link on the top of each task.
 
 ### Ready?
 
-➡️ Go forward to [Task 0](test00.md)
+➡️ Go forward to [Task 0](tasks/git-intro.md)
```
Comment (dmk1111) [inline README.md]: `Task 0` name wouldn't remove problem, described in the issue. Probably we should get rid of numbering here as well.

Comment (viktor-yakubiv) [reply README.md]: I thought about it too but I was not sure is it necessary. We still need enumeration of tasks to make it clear for students.

**Comment by viktor-yakubiv** [comment]
I'm going to fix issue by @dmk1111 later in next PR. Please merge to avoid merge conflicts because of renaming files.

**Comment by OleksiyRudenko** [comment]
@viktor-yakubiv looks like we lost improvements from #98 and #100 
Can you please check and reproduce changes from #98 and restore archive as per #100 ?
Otherwise important contributions get discarded for nothing...

**Pull request merged**